src/iden3comm/handlers/payment.ts

```diff
@@ -310,6 +312,12 @@ export class PaymentHandler
     paymentRequest: PaymentRequestMessage,
     ctx: PaymentRequestMessageHandlerOptions
   ): Promise<BasicMessage | null> {
+    if (
```

Comment:
A lot of code duplication. Can we do something like that:
```js
package common
mustNotExpired = (obj: {expires_time?: number})) => {
 if paymentRequest?.expires_time && paymentRequest.expires_time < Math.floor(Date.now() / 1000)
     throw new Error
 }
 ```
 Usage:
 ```
 common.MustNotExpired(offer)
 or
 common.MustNotExpired(proposal)
 ...
 ```
 My idea is have a utility function that consumes interface with optional field  expires_time and does some logic

Reply:
done